src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -354,7 +377,8 @@ public override bool ReturnsVoid
         {
             get
             {
-                return this.flags.ReturnsVoid;
+                flags.TryGetReturnsVoid(out bool value);
```

Comment:
>flags.TryGetReturnsVoid(out bool value); [](start = 16, length = 40)

It feels like we should assert that the ```TryGetReturnsVoid``` returns true, or should trigger calculation to get the actual value.

Reply:
That assert will likely fail when stopping in a debugger. And the behavior is unchanged from previously.

---
In reply to: [296896682](https://github.com/dotnet/roslyn/pull/36152#discussion_r296896682) [](ancestors = 296896682)